plugins/woocommerce-beta-tester/.distignore

```diff
@@ -22,3 +22,14 @@ changelog/
 woocommerce-beta-tester.zip
 node_modules/
 bin/
+
```

Comment:
Nice, as a follow-up, we could also build the zip with composer's `--no-dev` flag to build without dev dependencies, which would also speed up the zip build too.

The core plugin does this [here](https://github.com/woocommerce/woocommerce/blob/ed5836cf25538521a6e9fe53c0ba71e481b4184a/plugins/woocommerce/bin/build-zip.sh#L25), but I couldn't figure out how composer is used in beta tester's build 😅

Reply:
Sounds good! I'll take a look at it.